### PR TITLE
Cleanup patch for fresh start

### PR DESCRIPTION
**Pull request by debloper**
- Branding update
- Added docs content
- Removed old app contents
- Removed some bitrotted files
- Package config, build, deps updated

Signed-off-by: Soumya Deb <debloper@gmail.com>

### CODE REVIEW

**Comment by debloper** [comment]
@TimothyAsir @ktdreyer please review the spec file changes

File: package.spec [hunk 1]
```diff
@@ -0,0 +1,41 @@
+%define pkg_name tendrl_frontend
+%define pkg_version 0.0.1
+%define pkg_release 1
+
```
Comment (TimothyAsir) [inline package.spec]: Use package name, version, release directly like
Name: tendrl-frontend
version: 0.0.1
release: 1%{?dist}

and use %{name}

File: package.spec [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+Name: %{pkg_name}
+Version: %{pkg_version}
+Release: %{pkg_release}%{?dist}
+BuildArch: noarch
+Summary: GUI for Tendrl
+License: ASL 2.0
+Group:   Applications/System
+Source0: %{pkg_name}-%{pkg_version}.tar.gz
+BuildRoot: %{_tmppath}/%{pkg_name}-%{pkg_version}-%{pkg_release}-buildroot
+Url: https://github.com/Tendrl/tendrl_frontend
```
Comment (TimothyAsir) [inline package.spec]: Url -> URL

File: package.spec [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+Requires: tendrl
+
+%description
+Contains the JavaScript GUI content for the Tendrl front-end components
+(dashboard, login screens, administration screens)
+
+%prep
+%setup -n %{pkg_name}-%{pkg_version}
```
Comment (TimothyAsir) [inline package.spec]: no need to specify -n ..

File: package.spec [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+
+%install
+rm -rf $RPM_BUILD_ROOT
```
Comment (TimothyAsir) [inline package.spec]: rm -rf not required

File: package.spec [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+install -m 755 -d $RPM_BUILD_ROOT/%{_datadir}/Tendrl/webapp
+cp -r ./dist/* $RPM_BUILD_ROOT/%{_datadir}/Tendrl/webapp/
+
+%clean
+echo "clean"
```
Comment (TimothyAsir) [inline package.spec]: remove echo

File: package.spec [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+
+%prep
+%setup -n %{pkg_name}-%{pkg_version}
+
+%install
+rm -rf $RPM_BUILD_ROOT
+install -m 755 -d $RPM_BUILD_ROOT/%{_datadir}/Tendrl/webapp
+cp -r ./dist/* $RPM_BUILD_ROOT/%{_datadir}/Tendrl/webapp/
+
+%clean
```
Comment (TimothyAsir) [inline package.spec]: clean not required

File: package.spec [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+Contains the JavaScript GUI content for the Tendrl front-end components
+(dashboard, login screens, administration screens)
+
+%prep
+%setup -n %{pkg_name}-%{pkg_version}
+
+%install
+rm -rf $RPM_BUILD_ROOT
+install -m 755 -d $RPM_BUILD_ROOT/%{_datadir}/Tendrl/webapp
+cp -r ./dist/* $RPM_BUILD_ROOT/%{_datadir}/Tendrl/webapp/
```
Comment (TimothyAsir) [inline package.spec]: remove / after $RPM_BUILD_ROOT or before %{_datadir}

File: package.spec [hunk 8]
```diff
@@ -0,0 +1,41 @@
...
+%description
+Contains the JavaScript GUI content for the Tendrl front-end components
+(dashboard, login screens, administration screens)
+
+%prep
+%setup -n %{pkg_name}-%{pkg_version}
+
+%install
+rm -rf $RPM_BUILD_ROOT
+install -m 755 -d $RPM_BUILD_ROOT/%{_datadir}/Tendrl/webapp
```
Comment (TimothyAsir) [inline package.spec]: I hope execute permission is not required

**Pull request merged**